resources/css/bem/profile-badges.less

```diff
@@ -26,4 +26,8 @@
     height: 40px;
     object-fit: contain;
   }
+
+  &__date {
+    color: @osu-colour-l1;
```

Comment:
use the css var version instead `hsl(var(--hsl-l1))`

The less variable `@osu-colour-*`versions are legacy variables